src/app/shared/dynamic-form/components/form-select/form-select.component.html

```diff
@@ -19,7 +20,9 @@
         {{ opt.value }}
       </mat-option>
     </mat-select>
-    <mat-hint *ngIf="config.hint">{{ config.hint || '' | translate }}</mat-hint>
+    <mat-hint class="hint" *ngIf="config.hint && !config.dontShowHint">{{
```

Comment:
why the need for an extra flag?
just don't pass hint if you don't want to show it

Reply:
We need the extra flag because we need to make sure we can hide it based on the user's email address. It should be shown by default, but if you select another user from the dropdown, we need to set this flag to hide the hint. in case if it has email we need to set it back and show it.

Reply:
@danielrus is right. We do not need another flag for this. Make use of the hint attribute that is already implemented

Reply:
Changed that.